__mocks__/react-native.js

```diff
@@ -15,7 +14,12 @@ jest.doMock('react-native', () => {
     // runs against index.native.js source and so anything that is testing a component reliant on withWindowDimensions()
     // would be most commonly assumed to be on a mobile phone vs. a tablet or desktop style view. This behavior can be
     // overridden by explicitly setting the dimensions inside a test via Dimensions.set()
-    let dimensions = CONST.TESTING.SCREEN_SIZE.SMALL;
+    let dimensions = {
```

Comment:
I had to make these changes because this mock file was importing `CONST` file which was in turn importing `src/libs/Url.js` file in which we have imported the polyfill. Since the polyfill import internally imports RN packages and since we are in test environment the packages were being used before mocking them (which is done later in this file) and the test suite broke.To fix this we have multiple options:

1. We can import the polyfill in the root `App.js` file as mentioned in the usage guidelines (https://github.com/charpeni/react-native-url-polyfill) but it will then override `URL` and `URLSearchParams` classes in the entire project.I actually committed that change in the previous commit and everything was working fine including the test cases.Also these 2 classes are not used much so the chance of breaking something is low and since the overrides basically extends the default RN classes so I think we are fine using this option too.

2. Remove the import from the test suite before the mocking happens.In this option I had 2 further options:

    - Remove the Url.js import from CONST.js as it was imported to use the [addTrailingForwardSlash](https://github.com/Expensify/App/blob/b94b48f0db315391bd4d781a6ca0940e70f8ac29/src/libs/Url.js#L7-L11) just at one place in the file.Also importing lib files in CONST file does not make sense.
    - The other option was to remove the CONST file import in this file and that's what I have done now.
    
Let me know what you think is the best way forward.

Reply:
> remove the CONST file import in this file

Current solution you did works for me.
@francoisl what do you think?

Reply:
Yeah works for me too.